source/module_base/lapack_connector.h

```diff
@@ -224,6 +238,31 @@ class LapackConnector
         delete[] aux;
         delete[] bux;
     }
+    // wrap function of fortran lapack routine zhegv.
+    static inline
+    void zhegv(	const int itype, const char jobz, const char uplo, const int n, std::complex<double>* a,
+                const int lda, const std::complex<double>* b, const int ldb, double* w, std::complex<double>* work,
+                int lwork, double* rwork, int info, int ld_real)
+    {	// Transpose the std::complex matrix to the fortran-form real-std::complex array.
+        std::complex<double>* aux = LapackConnector::transpose(a, n, lda, ld_real);
+        std::complex<double>* bux = LapackConnector::transpose(b, n, ldb, ld_real);
```

Comment:
Maybe storing the array in column major to avoid transposing should be considered.

Reply:
Since the data in the entire process of the current CG is stored in column major, this is only to fit the whole process. 
Perhaps, it will be done later to replace all data in the entire CG process with column priority storage to avoid transpose operations, but this should not be the focus of this PR.